publisher-sdk/src/main/java/com/criteo/publisher/Session.kt

```diff
@@ -21,9 +21,9 @@ internal class Session(private val clock: Clock) {
   private val startingTime = clock.currentTimeInMillis
 
   /**
-   * Returns the current time in milliseconds since the initialization of the SDK.
+   * Returns the current time in seconds since the initialization of the SDK.
    *
-   * @return the difference, measured in milliseconds, between the current time and the SDK init.
+   * @return the difference, measured in seconds, between the current time and the SDK init.
    */
-  fun getDuration(): Long = clock.currentTimeInMillis - startingTime
+  fun getDurationInSeconds(): Long = (clock.currentTimeInMillis - startingTime) / 1000
```

Comment:
Nit: I guess we can use a simple `Int` then

Reply:
Done